### ISSUE TITLE
Add a Consortium type

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#1559]
This was originally suggested via the #1495 proposal of LibrarySystem but has broader use, and its use for LibrarySystem may not be a good fit.

**Comment by danbri** [comment]
See https://github.com/schemaorg/schemaorg/pull/1539 for examples

**Comment by jhabdas** [comment]
I haven't looked through the examples yet but this type looks perfect for Chicago Gang History, where Gangs are marked up as Organizations and gang alliances become Consortiums. Planning to move forward with the WIP examples and will refine later once pulled into pseudo-standards for Schema.org.

**Comment by jhabdas** [comment]
I saw there was already some extensive conversation in #1495 and it's hard to follow. It would be helpful to spell out the key benefits of a new Consortium type based on the conversation as it seems as though parent and sub-orgs are generic enough to handle this use case—though like I said there's quite a bit of detailed discussion which would need to be absorbed from the linked issue in order to understand the true benefit of adding a new type.

**Comment by thadguidry** [comment]
Circling back to this... we miss placed the common use case itself, not that of any legal relationships, but instead "to collaborate or share resources among members".

Wikidata has the definition being "A consortium is an association of two or more individuals, companies, organizations or governments (or any combination of these entities) with the objective of participating in a common activity or pooling their resources for achieving a common goal". - and I highly suggest reading [https://en.wikipedia.org/wiki/Consortium](https://en.wikipedia.org/wiki/Consortium)  The Latin etymology is interesting.

For profit example:
Hulu, the American video streaming service, is owned by a consortium of large media conglomerates including Time Warner, 21st Century Fox, Comcast, and the Walt Disney Company.

Non profit example:
World Wide Web Consortium (W3C), is a consortium that standardizes web technologies like HTML, XML and CSS.

- [ ] We need to nail down a much better definition for our [Consortium](https://pending.schema.org/Consortium) type in pending.

I would much rather see our [Consortium](https://pending.schema.org/Consortium) type (which can be used for profit, like Banks, and non-profit organizations, like Libraries) mention something about "collaborating or sharing" that to cover it all, something like:

_"A membership Organization whose members collaborate or pool their resources together to achieve a common goal, and which might involve the ideas of coopetition, rather than competition, for the strategic benefit between all members.  Some members still might compete in other areas."_

**Comment by RichardWallis** [comment]
I agree that the current description is a little lacking.

Your suggestion and reference to Wikipedia are a good start to its improvement.  I think references to strategic benefit and and competition may unnecessarily raise questions for those thinking of applying it.

How about:

_A membership Organization of two or more individuals or organizations of any type, who collaborate with the objective of participating in a common activity or pooling their resources for achieving a common goal._
